contracts/attesters/hydra-s1/HydraS1AccountboundAttester.sol

```diff
@@ -0,0 +1,289 @@
+
+import {IHydraS1AccountboundAttester} from './interfaces/IHydraS1AccountboundAttester.sol';
+import {HydraS1SimpleAttester} from './HydraS1SimpleAttester.sol';
+import {Ownable} from '@openzeppelin/contracts/access/Ownable.sol';
+
+// Core protocol Protocol imports
+import {Request, Attestation, Claim} from '../../core/libs/Structs.sol';
+import {IAttester} from '../../core/Attester.sol';
+
+// Imports related to HydraS1 Proving Scheme
+import {HydraS1Base, HydraS1Lib, HydraS1ProofData, HydraS1ProofInput, HydraS1Claim} from './base/HydraS1Base.sol';
+
+/**
+ * @title  Hydra-S1 Accountbound Attester
+ * @author Sismo
+ * @notice This attester is part of the family of the Hydra-S1 Attesters.
+ * Hydra-S1 attesters enable users to prove they have an account in a group in a privacy preserving way.
+ * The Hydra-S1 Simple Attester contract is inherited and holds the complex Hydra S1 verification logic.
+ * Request verification alongside proof verification is already implemented in the inherited HydraS1SimpleAttester, along with the buildAttestations logic.
+ * However, we override the buildAttestations function to encode the nullifier and its burn count in the user attestation.
+ * The _beforeRecordAttestations is also overriden to fit the Accountbound logic.
+ * We invite readers to refer to:
+ *    - https://hydra-s1.docs.sismo.io for a full guide through the Hydra-S1 ZK Attestations
+ *    - https://hydra-s1-circuits.docs.sismo.io for circuits, prover and verifiers of Hydra-S1
+
+ * This specific attester has the following characteristics:
+
+ * - Zero Knowledge
+ *   One cannot deduct from an attestation what source account was used to generate the underlying proof
+
+ * - Non Strict (scores)
+ *   If a user can generate an attestation of max value 100, they can also generate any attestation with value < 100.
+ *   This attester generate attestations of scores
+
+ * - Nullified
+ *   Each source account gets one nullifier per claim (i.e only one attestation per source account per claim)
+ *   While semaphore/ tornado cash are using the following notations: nullifierHash = hash(IdNullifier, externalNullifier)
+ *   We prefered to use the naming 'nullifier' instead of 'nullifierHash' in our contracts and documentation.
+ *   We also renamed 'IdNullifier' in 'sourceSecret' (the secret tied to a source account) and we kept the 'externalNullifier' notation.
+ *   Finally, here is our notations at Sismo: nullifier = hash(sourceSecret, externalNullifier)
+
+ * - Accountbound (with cooldown period)
+ *   Users can choose to delete or generate attestations to a new destination using their source account.
+ *   The attestation is "Accountbound" to the source account.
+ *   When deleting/ sending to a new destination, the nullifier will enter a cooldown period, so it remains occasional.
+ *   The duration of the cooldown period is different for each group, if the cooldown duration = 0 then the group does not allow accountbound attestations.
```

Comment:
Say more clearly: 
Each group has its own cooldown duration, set by the admin of the attester
A group that has its cooldown udration set to 0 means it is a group which does not feature accountbound attestations, attestations can not be transferred

Reply:
Say more clearly: 
Each group has its own cooldown duration, set by the admin of the attester
A group that has its cooldown udration set to 0 means it has been configured to not feature accountbound attestations, attestations can not be transferred